### ISSUE TITLE
Automação + Issues

### ISSUE THREAD

**Issue by onhernandes** [woliveiras/nodejs-in-brazil#5]
Que tal inserir as coisas(projetos, empresas, perfis) por issues, separar cada um por uma issue e montar um pequeno bot (em NodeJS claro hahahaha) pra ler as issues, montar um template e colocar no lugar certo? Acho melhor pra quem for inserir e pra você @woliveiras não ter que ficar manuseando PRs a torto e a direito. Que tal?

**Comment by onhernandes** [comment]
Me prontifico a criar esse bot :p sempre quis mexer com a API do Github hahahaha

**Comment by JonasUliana** [comment]
Concordo, seria muito mais fácil para mapeação dos profissionais aqui no BR!

**Comment by lagden** [comment]
Também acho interessante colocar os módulos e libs...
mas é necessário um índice para categorizar melhor...
o do @sindresorhus é bem bacana, podemos nos inspirar!!!

**Comment by onhernandes** [comment]
@lagden isso cai nos projetos, lá abrange tanto framworks quanto libs e outros projetos

**Comment by lagden** [comment]
@onhernandes entendo...
mas precisamos pensar uma maneira de quebrar mais ainda...
algo semelhante na biologia, por exemplo... `Domínio -> Reino -> Filo -> Divisão -> Classe -> Ordem -> Família -> Género -> Espécie -> Subespécie...`

Por exemplo... eu mandei um PR com um projeto:
 - https://github.com/lagden/cep-koa-api

mas o projeto acima utiliza a lib:
- https://github.com/lagden/cep

Os dois são projetos, mas um é uma API e o outro apenas uma lib...

Sacou?! Deu pra pegar a idéia da organização e ordenação das coisas?

**Comment by woliveiras** [comment]
PUTZ! QUE IDEIA FERA!

Acho super válido e vai ajudar muito se você fizer esse esquema, ein @onhernandes.

Se quiser se envolver em outro projeto, da uma olhada: https://github.com/training-center/mentoria/issues/830

**Comment by woliveiras** [comment]
Baseado na estrutura que temos hoje, você consegue pensar em uma árvore legal, @lagden?

**Comment by onhernandes** [comment]
@woliveiras vou montar um esqueleto +/- e depois mando um PR. Assim que tivermos uma árvore legal, podemos pensar sobre quais labels usar pra identificar.
